README.MD

```diff
@@ -44,6 +44,8 @@ Mod-inventory-storage implements domain event pattern and requires kafka to be l
 `localhost:9092`. You can override kafka port and host by setting `KAFKA_PORT` and
 `KAFKA_HOST` environment variables.
 
+Please note that the test suite will fail, however, if kafka is listening to `localhost:9092` during `mvn install` or `mvn test`. 
```

Comment:
It shouldn't fail, we start embedded kafka during testing. 

Could you provide logs?

Reply:
I wonder if this is connected to the mod-inventory build also failing Kafka related tests for me. I don't have Kafka running locally.

Reply:
I followed Apache's kafka install instructions at https://kafka.apache.org/quickstart

And checked that nothing ran on 9092, yet
root@NE-T460:/home/ne# sudo lsof -i :9092
root@NE-T460:/home/ne# 
until I start the server as per Apache's instructions:
root@NE-T460:/home/ne# sudo lsof -i :9092
root@NE-T460:/home/ne# sudo lsof -i :9092
COMMAND  PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
java    9586   ne  170u  IPv6  84501      0t0  TCP *:9092 (LISTEN)
java    9586   ne  189u  IPv6  82707      0t0  TCP localhost:44190->NE-T460:9092 (ESTABLISHED)
java    9586   ne  190u  IPv6  84503      0t0  TCP NE-T460:9092->localhost:44190 (ESTABLISHED)
root@NE-T460:/home/ne# 

I then run mvn test  for mod-inventory-storage   but the result is:

Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 9.795 sec <<< FAILURE! - in org.folio.rest.api.StorageTestSuite
org.folio.rest.api.StorageTestSuite  Time elapsed: 9.795 sec  <<< ERROR!
org.apache.kafka.common.KafkaException: Socket server failed to bind to localhost:9092: Address already in use.
	at org.folio.rest.api.StorageTestSuite.before(StorageTestSuite.java:152)

Reply:
The problem is, though, that this was but one test run. I seem to get different outcomes each time I run the current test suite for mod-inventory-storage, so have a hard time providing you with a reliably reproducible test run.

Reply:
Ok, got it, thanks. So the tests uses fixed port for the embedded kafka (9092), which is busy because of the 'real' kafka deployed. 

We should use a random port for tests instead. I have created https://issues.folio.org/browse/MODINVSTOR-678